app/main/appium.js

```diff
@@ -60,9 +78,12 @@ function connectStartServer (win) {
     args.throwInsteadOfExit = true;
 
     // set up our log watcher
-    logWatcher = setInterval(() => {
+    logWatcher = setInterval(async () => {
       if (batchedLogs.length) {
-        win.webContents.send('appium-log-line', batchedLogs);
+        try {
+          await fs.writeFile(logFile, batchedLogs.map((log) => `[${log.level}] ${log.msg}`).join('\n'), {'flag': 'a'});
```

Comment:
this would look better on multiple lines. one line for the map, the other for the write.